includes/admin/payments/orders.php

```diff
@@ -881,42 +996,55 @@ function edd_order_details_attributes( $order ) {
 					<label for="edd-payment-date" class="label">
 						<?php esc_html_e( 'Date', 'easy-digital-downloads' ); ?>
 					</label>
-					<input type="text" id="edd-payment-date" name="edd-payment-date" value="<?php echo esc_attr( $order_date->format( 'Y-m-d' ) ); ?>" class="medium-text edd_datepicker" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>"/>
+					<input type="text" id="edd-payment-date" name="edd-payment-date" value="<?php echo esc_attr( $order_date->format( 'Y-m-d' ) ); ?>" class="edd_datepicker" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>"/>
 
 				</div>
 
 				<div class="edd-admin-box-inside">
-					<span class="label">
-						<?php echo esc_html( __( 'Time', 'easy-digital-downloads' ) . ' (' . edd_get_timezone_abbr() . ')' ); ?>
-					</span>
-
-					<label for="edd_payment_time_hour" class="screen-reader-text">Hour</label>
-					<?php
-					echo EDD()->html->select( array(
-						'name'             => 'edd-payment-time-hour',
-						'id'               => 'edd-payment-time-hour',
-						'options'          => edd_get_hour_values(),
-						'selected'         => $order_date->format( 'H' ),
-						'chosen'           => true,
-						'class'            => 'edd-time',
-						'show_option_none' => false,
-						'show_option_all'  => false,
-					) ); // WPCS: XSS ok.
-					?>
-					:
-					<label for="edd_payment_time_min" class="screen-reader-text">Minute</label>
-					<?php
-					echo EDD()->html->select( array(
-						'name'             => 'edd-payment-time-min',
-						'id'               => 'edd-payment-time-min',
-						'options'          => edd_get_minute_values(),
-						'selected'         => $order_date->format( 'i' ),
-						'chosen'           => true,
-						'class'            => 'edd-time',
-						'show_option_none' => false,
-						'show_option_all'  => false,
-					) ); // WPCS: XSS ok.
-					?>
+					<fieldset class="edd-form-group">
+						<legend class="edd-form-group__label">
+							<?php echo esc_html( __( 'Time', 'easy-digital-downloads' ) . ' (' . edd_get_timezone_abbr() . ')' ); ?>
+						</legend>
+
+						<div class="edd-form-group__control">
+							<label for="edd-payment-time-hour" class="screen-reader-text">
```

Comment:
Because the `EDD()->html->select()` method converts IDs to use underscores instead of dashes, the label `for` should be `edd_payment_time_hour`, I think. Same for the minutes label. The ID for the hour doesn't _have_ to change but could, just so it's consistent with this label.